src/app/visual/phone-field/phone-field-visual.component.html

```diff
@@ -10,6 +10,7 @@
   [@.disabled]="true"
 >
   <sky-phone-field
+    [allowExtensions]='false'
     [returnFormat]="'international'"
```

Comment:
I know this isn't part of your current changes, but I just noticed this is using input binding to assign a literal value, and I think this line probably led to the accidental single quotes above. Do you mind changing it to `returnFormat="international"`?

Reply:
Fixed